src/components/Table/GridTable.tsx

```diff
@@ -778,6 +778,8 @@ export type GridColumn<R extends Kinded, S = {}> = {
    * - The default value is `1fr`
    */
   w?: number | string;
+  /** The minimum width the column can shrink to */
+  mw?: string;
```

Comment:
Should we enforce that this can only be px or percent?

Reply:
Yea, probably should. Otherwise if we use something like `fit-content` then it allows for cells in a column to be drawn at different widths. Will add that in.